docs/template_plugin/tests/functional/op_reference/select.cpp

```diff
@@ -0,0 +1,139 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+#include <tuple>
+
+#include "base_reference_test.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+
+struct SelectParams {
+    template <class IT, class OT>
+    SelectParams(const ngraph::element::Type& data_type, const ngraph::op::AutoBroadcastSpec& broadcast, const ngraph::PartialShape& select_input_pshape,
```

Comment:
Above is `using namespace ngraph;`. Why types are prefix by `ngraph::`?

Reply:
It's unnecessary, removed.